### DISCUSSION TITLE
upgrading filament from version 2 to version 3

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#8483] [Help]
when upgrading filament frfom filament 2 to version 3 
i ran this one : 
composer require filament/upgrade:"^3.0-stable" -W --dev   and i got a success message ,afterwards i ran this

php artisan filament:install  >> but i got the message : command doesn't exist 
![Screenshot from 2023-09-14 09-58-01](https://github.com/filamentphp/filament/assets/56083879/e6169d4c-8f5d-41f1-972e-c0486583f5ff)

**Comment by leandrocfe** [comment]
Did you run `vendor/bin/filament-v3` ?
